packages/back-end/src/services/organizations.ts

```diff
@@ -273,6 +275,59 @@ export async function addMemberToOrg({
   });
 }
 
+export async function addMemberToTeam({
+  organization,
+  userId,
+  teamId,
+}: {
+  organization: OrganizationInterface;
+  userId: string;
+  teamId: string;
+}): Promise<void> {
+  // If member is a pending member, skip
+  if (organization?.pendingMembers?.find((m) => m.id === userId)) {
+    return;
+  }
+
+  const member = organization.members.find((m) => m.id === userId);
+
+  // If member doesn't exist in the org or is already in the team, skip
+  if (!member || member.teams?.includes(teamId)) {
+    return;
+  }
+
+  // Create teams array for member if it does not exist
+  if (!member.teams) {
+    member.teams = [];
+  }
+  member.teams.push(teamId);
+
+  await updateOrganization(organization.id, { members: organization.members });
+}
+
+export async function removeMemberFromTeam({
+  organization,
+  userId,
+  teamId,
+}: {
+  organization: OrganizationInterface;
+  userId: string;
+  teamId: string;
+}): Promise<void> {
+  const member = organization.members.find((m) => m.id === userId);
+
+  // If member doesn't exist in the org or isn't in the team, skip
+  if (!member || !member.teams?.includes(teamId)) {
+    return;
+  }
+
+  const indexToDelete = member.teams.indexOf(teamId);
+
+  member.teams.splice(indexToDelete, indexToDelete);
```

Comment:
I think the second argument here needs to be `1` rather than `indexToDelete` as the second argument controls the `deleteCount`.